CMake/RealmCore.cmake

```diff
@@ -390,8 +390,21 @@ macro(build_realm_sync)
     set_property(TARGET realm-sync-server PROPERTY IMPORTED_LOCATION_RELEASE ${sync_server_library_release})
     set_property(TARGET realm-sync-server PROPERTY IMPORTED_LOCATION ${sync_server_library_release})
 
-    find_package(PkgConfig)
-    pkg_check_modules(YAML QUIET yaml-cpp)
+    if(REALM_PLATFORM STREQUAL "Android")
+      find_host_package(Git)
+      ExternalProject_Add(YAML
```

Comment:
`YAML` -> `yaml`? We always use lower cases for external project name in this file.